src/AccessibilityInsights.SharedUx/Controls/EventConfigurationControl.xaml.cs

```diff
@@ -235,6 +235,7 @@ public void SetLists(List<RecordEntitySetting> list, List<RecordEntitySetting> s
 
         private void UpdateListViews()
         {
+#pragma warning disable CA1508 // Dead code warning doesn't apply here
```

Comment:
Analyzer says that we're checking for null in the lines that new up the ListViewSelectionLock objects, but I don't believe it